scripts/scil_bundle_label_map.py

```diff
@@ -281,9 +280,6 @@ def main():
     logging.debug(
         f'Trim bundle(s) in {round(time.time() - timer, 3)} seconds.')
 
-    # Use later to trim the streamlines without assignement
-    min_len = np.min(length(concat_sft.streamlines))
```

Comment:
If the streamlines labels don't span the shortest streamlines, no labelled streamlines would be saved.